### PR TITLE
feat: redesigning pdfeditor with design tokens

### PR DESCRIPTION
**Pull request by RamaChandiniDevarapalli**

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts [hunk 1]
```diff
@@ -237,37 +252,35 @@ export const PdfEditorLabelWrapper = styled.div`
   .badge {
     margin: 0px 0px 0px 3px;
   }
-`;
+`; // no spacing design token for 4.5rem and 3px
```
Comment (rstyan) [inline apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts:255]: If the Figma design does't specify a token, then lets not worry about it and just do what you did, but leave the comment out.

Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts:255]: I did commented these before figma design I will remove them. Sorry somehow i missed to remove comments

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts [hunk 2]
```diff
@@ -169,60 +181,63 @@ export const BodyGlobalStyles = createGlobalStyle<{ hideOverflow: boolean }>`
...
+  min-height: var(--goa-spacing-xl);
+  margin: 0;
 `;
 
 export const EditTemplateActions = styled.div`
   display: flex;
   justify-content: left;
-  gap: 1rem;
-  margin: 1rem 0 2rem 0;
+  // gap: var(--goa-spacing-m);
```
[the remark this answers is not in the document]
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts:215]: removed

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts [hunk 3]
```diff
@@ -169,60 +181,63 @@ export const BodyGlobalStyles = createGlobalStyle<{ hideOverflow: boolean }>`
   }
 `;
 export const ModalContent = styled.div`
-  background: white;
+  // background: white;
```
[the remark this answers is not in the document]
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts:184]: removed

File: apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx [hunk 4]
```diff
@@ -262,7 +262,7 @@ export const TemplateEditor = ({ errors }: TemplateEditorProps): JSX.Element =>
                     <GoAButton
                       type="secondary"
                       testId="pdf-delete-all-files"
-                      size="compact"
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/TemplateEditor.tsx:265]: Why change size of the button, it not related with design token?

**Pull request merged**